src/sentry/migrations/0185_group_category.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 1.11.29 on 2021-04-08 06:32
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.foreignkey
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
```

Comment:
Because we're creating a foreign key with group, we'll want to do one of two things here

- mark as dangerous, and try to run this when we would actually be able to acquire a lock (unlikely given that its with the Groups Table)
- don't add this foreign key constraint at the DB level, by setting `db_constraint=False`

Reply:
I second the `db_constraint=False` suggestion